### PR TITLE
Podcast show final touches

### PR DESCRIPTION
**Pull request by takeller**
- Fixes issue with collapse datatargets being unique
- Changes tag color
- Fixes error with social_media method
- Increases location size

### CODE REVIEW

File: app/views/podcasts/show.html.erb [hunk 1]
```diff
@@ -24,20 +24,21 @@
 </section>
 
 <section class='episodes'>
-  <% @podcast_facade.episodes.each do |episode| %>
+  <h2>Episodes</h2>
+  <% @podcast_facade.episodes.each_with_index do |episode, index| %>
       <p class="card-text">
-        <button class="btn btn-outline btn-xl" type="button" data-toggle="collapse" data-target=<%= "#episode#{episode.title.delete(' ')[0..5]}" %> aria-expanded="false" aria-controls=<%= "episode#{episode.title.delete(' ')[0..5]}" %>>
+        <button class="btn btn-outline btn-xl col-sm-5" type="button" data-toggle="collapse" data-target=<%= "#episode#{index}" %> aria-expanded="false" aria-controls=<%= "episode#{index}" %>>
```
Comment (muydanny) [inline app/views/podcasts/show.html.erb:30]: Nice!

Comment (kiverso) [reply app/views/podcasts/show.html.erb:30]: looks good with the fixed width

File: app/views/podcasts/show.html.erb [hunk 2]
```diff
@@ -24,20 +24,21 @@
...
       </p>
-      <div class="collapse" id=<%= "episode#{episode.title.delete(' ')[0..5]}" %>>
+      <div class="collapse" id=<%= "episode#{index}" %>>
         <div class="card card-body episode-card">
           <div class="card text-white episode-card mb-3 w-75">
             <div class="card-body">
               <h5 class="card-title"><%= raw(episode.title).html_safe %></h5>
               <p class='card-text'><%= raw(episode.description).html_safe %></p>
               <audio src= <%= episode.audio %> controls> </audio>
-              <div class="card-footer text-muted episode-footer"><%= episode.pub_date %></div>
```
Comment (kiverso) [inline app/views/podcasts/show.html.erb:40]: good catch, this stands out much better on the page now.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -654,7 +654,8 @@ footer ul li a:hover, footer ul li a:focus, footer ul li a:active, footer ul li
 .tags {
   padding-top: 0;
   padding-bottom: 0;
-  color: blue;
+  color: #fdcc52;
```
Comment (kiverso) [inline app/assets/stylesheets/application.scss:657]: this is a much easier color to read and matches the rest of the app.

**Pull request merged**